src/GafferImage/OpenImageIOReader.cpp

```diff
@@ -523,6 +523,19 @@ void OpenImageIOReader::hashFormat( const GafferImage::ImagePlug *output, const
 	hashFileName( context, h );
 	refreshCountPlug()->hash( h );
 	missingFrameModePlug()->hash( h );
+
+	std::string fileName = fileNamePlug()->getValue();
+	MissingFrameMode mode = (MissingFrameMode)missingFrameModePlug()->getValue();
```

Comment:
As noted below, I think we should be able to remove the `missingFrameModePlug()->hash()` above since we're actually computing and using the value here.